### PR TITLE
feat(sanitize): user can self xss

### PR DESCRIPTION
**Pull request by dalipkumar703**
This PR fix user can self-xss themselves bug in which sending <iframe> doesn't work.
JIRA: https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-137465

### CODE REVIEW

File: src/components/ComposerQuill/sanitize.js [hunk 1]
```diff
@@ -24,6 +24,10 @@ function sanitizerPlugin(md) {
           }
         });
       }
+      if (type === 'html_block') {
+        // eslint-disable-next-line no-param-reassign
+        token.content = token.content.replace(/</g, '&lt;').replace(/(?!^)>/gm, '&gt;');
```
Comment (holsted) [inline src/components/ComposerQuill/sanitize.js:29]: @phatv can you weigh in on this?

Comment (phatv) [reply src/components/ComposerQuill/sanitize.js:29]: This looks fine to me. `html_block` might have element tags in it that we don't want to actually display as elements but instead as the carets.

But would you mind sharing what the text was for it to come in as html_block?

Comment (dalipkumar703) [reply src/components/ComposerQuill/sanitize.js:29]: it is ``<iframe onload=alert(document.cookie) %00 src="<h1>Injected H1</h1>"%00>`` that will come inside html_block .

**Pull request merged**